catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/MetacardCreator.java

```diff
@@ -68,24 +69,77 @@ public static Metacard createMetacard(final Metadata metadata, final String id,
             metacard.setAttribute(new AttributeImpl(Metacard.MODIFIED, modifiedDate));
         }
 
-        if (StringUtils.isNotBlank(id)) {
-            metacard.setAttribute(new AttributeImpl(Metacard.ID, id));
-        }
-
-        if (StringUtils.isNotBlank(metadataXml)) {
-            metacard.setAttribute(new AttributeImpl(Metacard.METADATA, metadataXml));
-        }
+        setAttribute(metacard, Metacard.ID, id);
 
+        setAttribute(metacard, Metacard.METADATA, metadataXml);
         final String lat = metadata.get(Metadata.LATITUDE);
         final String lon = metadata.get(Metadata.LONGITUDE);
-        final String wkt = toWkt(lon, lat);
-        if (StringUtils.isNotBlank(wkt)) {
-            metacard.setAttribute(new AttributeImpl(Metacard.GEOGRAPHY, wkt));
-        }
+        setAttribute(metacard, Metacard.GEOGRAPHY, toWkt(lon, lat));
+
+        setAttribute(metacard,
```

Comment:
The tika format shouldn't be mapped to datatype. Datatype has a specific [enumeration](https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/definitions/core-attributes-validator.json#L3).  

I think the appropriate mapping here is to [media.format](https://codice.atlassian.net/wiki/display/DDF/media.format).